docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -408,25 +408,6 @@ print(f">> dynamical decoupling sequence to use: {sampler.options.dynamical_deco
 ```    
   </TabItem>
 
-  <TabItem value="SamplerV1" label="Sampler (V1)">
-    ```python
-from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
-
-service = QiskitRuntimeService()
-backend = service.least_busy(operational=True, simulator=False)
-
-# Setting options during primitive initialization
-options = Options()
-# This uses auto complete.
-options.resilience_level = 2 
-sampler = Sampler(backend=backend, options=options)
-
-# Setting options after primitive initialization.
-# This does bulk update.
-sampler.set_options(shots=2000, optimization_level=0)
-  ```  
-  </TabItem>
-
   <TabItem value="SamplerV1" label="Sampler (V1)">
```

Comment:
Why is the V1 tab so much shorter than the V2 tab? Only double checking this is the correct content.

Reply:
The V2 tab calls the service and defines the backend, but it doesn't really need to so I deleted those lines.